shell.nix

```diff
@@ -29,6 +29,8 @@ with packageSet; haskell.packages.shellFor {
     dev.packages.purty
     dev.packages.purs
     dev.packages.spago
+    # From haskell.nix
+    (pkgs.rWrapper.override { packages = with pkgs.rPackages; [tidyverse dplyr stringr MASS]; } )
```

Comment:
Hm, I'd have hoped this got pulled in as a build dep by the shell :thinking:

Reply:
I _think_ it didn't work, I could try again.

Reply:
Works.